test/c3/appframework/appframework_test.go

```diff
@@ -515,27 +573,29 @@ var _ = Describe("c3appfw test", func() {
 			testenv.VerifyRFSFMet(deployment, testenvInstance)
 
 			// Verify apps with local scope are downloaded by init-container
+			appVersion := "V1"
```

Comment:
Is this variable only used in log messages?

Reply:
yes. Just for log messages. Since we do both upgrade and downgrades within the same test case. It was getting confusing understanding if the app install checks were for which in cases of failures